.github/workflows/update-lockfile.yml

```diff
@@ -32,21 +38,20 @@ jobs:
           # Pretty print
           cp lockfile.json /tmp/lockfile.json
           cat /tmp/lockfile.json | jq --sort-keys > lockfile.json
-
-      - name: Commit changes
-        run: |
-          git config user.name 'GitHub'
-          git config user.email 'noreply@github.com'
-          git add lockfile.json
           UPDATED_PARSERS=$(/tmp/jd -f merge /tmp/old_lockfile.json lockfile.json | jq -r 'keys | join(", ")')
           echo "UPDATED_PARSERS=$UPDATED_PARSERS" >> $GITHUB_ENV
-          git commit -m "Update parsers: $UPDATED_PARSERS" || echo 'No commit necessary!'
-          git clean -xf
 
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v5
         with:
+          token: ${{ steps.app-token.outputs.token }}
+          commit-message: "Update parsers: ${{ env.UPDATED_PARSERS }}"
           title: "Update lockfile.json: ${{ env.UPDATED_PARSERS }}"
+          body: "[beep boop](https://github.com/peter-evans/create-pull-request)"
```

Comment:
Guessing the "beep boop" is intentional and not something you intended to remove? :)

Reply:
It is, although I'm not 100% I'll leave it in ;) (But I can't think of any more useful body, without adding more scripting for, e.g., changelogs.)

Reply:
Eh, I'm perfectly fine with it as is.